terraform/modules/deploy_hook/lambdas/hook.py

```diff
@@ -12,8 +12,8 @@
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
-gdx_url = os.environ["gdx_url"]
-events_url = gdx_url + "/events"
+test_gdx_url = os.environ["test_gdx_url"]
```

Comment:
Should we expand "test" here, it's a smoke test specifically?

Reply:
not really sure we should, as this is what codedeploy calls the "test" connection. Are you thinking we will get confused with it being a test env url vs a test url for this env?

Reply:
I was worried about it in the abstract as a "what does the word test even mean", if it's consistent with CodeDeploy and naming across the AWS side of things it's reasonable to keep it as is

Reply:
it is, CodeDeploy call it the test listener